colosseum/parser.py

```diff
@@ -126,3 +128,33 @@ def color(value):
                 pass
 
     raise ValueError('Unknown color %s' % value)
+
+
+RECT_PATTERN = re.compile(r'''
+    # Check for item with commas
+    \s*rect\(\s*
+        ([0-9A-Za-z]+)\s*,\s*
+        ([0-9A-Za-z]+)\s*,\s*
+        ([0-9A-Za-z]+)\s*,\s*
+        ([0-9A-Za-z]+)
+    \s*\)\s*
+    |
+    # Check for item with spaces
+    \s*rect\(\s*
+        ([0-9A-Za-z]+)\s+
+        ([0-9A-Za-z]+)\s+
+        ([0-9A-Za-z]+)\s+
+        ([0-9A-Za-z]+)
+    \s*\)\s*''', re.VERBOSE | re.IGNORECASE)
+
+
+def rect(value):
+    """Parse a given rect shape."""
+    from .shapes import Rect
```

Comment:
This is here because the shape uses units, so to prevent circular imports.

I could not use shapes.Rect here but use it on validators and have the Rect import there, or I could move the Rect shape to this file.

Thoughts @freakboy3742 ?

Reply:
Ok will update